doc/interop-test-descriptions.md

```diff
@@ -90,34 +90,40 @@ Client asserts:
 * clients are free to assert that the response payload body contents are zero
   and comparing the entire response message against a golden response
 
-### large_compressed_unary
-
-This test verifies compressed unary calls succeed in sending messages. It
-sends one unary request for every payload type, with and without requesting a
-compressed response from the server.
-
-In all scenarios, whether compression was actually performed is determined by
-the compression bit in the response's message flags.
+### client_compressed_unary
 
+This test verifies the client can compress unary messages. It sends one
+unary request for a compressable payload type, with and without compression.
 
 Server features:
 * [UnaryCall][]
-* [Compressable Payload][]
-* [Uncompressable Payload][]
+* [Compressed Request][]
```

Comment:
Oh, it does, but with a different spelling: CompressedRequest

Ditto for Compressed Response elsewhere.